scout/demo/delivery_report.html

```diff
@@ -14,12 +13,11 @@
-  </style>
-
+</style>
 
-  <style>
+<style>
     @page {
-      size: A4;
```

Comment:
This is the actual fix for the content not fitting the PDF reports in #3108

Reply:
🥇